shaders/src/common_getters.fs

```diff
@@ -39,7 +39,7 @@ vec4 getResolution() {
 
 /** @public-api */
 vec3 getWorldCameraPosition() {
-    return frameUniforms.cameraPosition;
+    return frameUniforms.worldOffset;
```

Comment:
Why rename this UBO? It's still the camera position in world space right?

Reply:
Good point, reverted this piece.